WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -266,18 +275,25 @@ private void PerformUpdate(WtsModel wts, Version currentVersion, Version version
                     return;
                 }
             }
+
+            var subject = "WTS Auto Updater - Update installed";
+            var message = $"WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.";
             
-            logger.LogInformation($"WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.");
+            logger.LogInformation(message);
 
             if (wts.SendEmailOnUpdateComplete)
             {
-                EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Update installed", $"The service for WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.", wts.ServiceName);
+                InformPeople(wts,subject,message);
```

Comment:
I'm not going to reject the PR for this, but please separate parameters with spaces.